ui/apps/platform/src/Containers/VulnMgmt/List/Images/WatchedImagesDialog.tsx

```diff
@@ -131,7 +131,7 @@ const WatchedImagesDialog = ({ closeDialog }: WatchedImagesDialogProps): ReactEl
                     validationSchema={Yup.object({
                         imageTag: Yup.string()
                             .matches(
-                                /([a-z.]+\/)([a-z0-9-]+\/)([a-z0-9-@./]+)(?::[0-9a-z\-.]+)?/,
+                                /([a-z.]+\/)([a-z0-9-]+\/)?([a-z0-9-@.\/]+)(?::[0-9a-z\-.]+)?/,
```

Comment:
@clickboo style-checks reports error
134:71  error  Unnecessary escape character: \/  no-useless-escape

Is the difference:
* `([a-z.]+\/)` and `([a-z0-9-]+\/)?` have `/` outside `[…]` character class
* `([a-z0-9-@.\/]+)` has `/` inside `[…]` character class: which causes me to ask does its presence imply slashes can occur optionally within or at the end of image names?